tools/maxquant/maxquant.xml

```diff
@@ -364,11 +311,11 @@ sample:$j
     <expand macro="misc_conditional" />
   </inputs>
   <outputs>
-    <data format="tabular" name="output_protein_groups" label="MaxQuant Protein Groups for ${on_string}"/>
-    <data format="tabular" name="output_peptides" label="MaxQuant Peptides for ${on_string}"/>
-    <data format="tabular" name="output_evidence" label="MaxQuant Evidence for ${on_string}"/>
-    <data format="tabular" name="output_parameters" label="MaxQuant Tabular Parameters for ${on_string}"/>
-    <data format="tabular" name="output_msms" label="MaxQuant MSMS for ${on_string}"/>
+    <data format="csv" name="output_protein_groups" label="MaxQuant Protein Groups for ${on_string}"/>
```

Comment:
I found this one: 
```
SED-replacement with PowerShell
cat somefile.txt | %{$_ -replace "expression","replace"}
```
As far as I know maxquant can run under wine, isn't it? If so we need to support both worlds or advertise only one.